src/Tools/MicroBuild/Build.proj

```diff
@@ -13,7 +13,9 @@
     <ScriptArgs Condition="'$(PB_SkipTests)' != 'true'">$(ScriptArgs) -testDesktop</ScriptArgs>
     <ScriptArgs Condition="'$(PB_SignType)' == 'real'">$(ScriptArgs) -signType "real"</ScriptArgs>
     <ScriptArgs Condition="'$(PB_SignType)' == 'test'">$(ScriptArgs) -signType "test"</ScriptArgs>
-    <ScriptArgs Condition="$(PB_PublishType.Contains('vsts'))">$(ScriptArgs) -publish</ScriptArgs>
+    <ScriptArgs Condition="'$(PB_PublishType)' == 'vsts'">$(ScriptArgs) -publishType "vsts"</ScriptArgs>
```

Comment:
Should this just `publishType "$(PB_PublishType)"` so new values just pass through?